client/store/modules/repository/comments/getters.js

```diff
@@ -1,17 +1,24 @@
 import filter from 'lodash/filter';
 import orderBy from 'lodash/orderBy';
 
-export const getActivityComments = state => activityId => {
-  const activityComments = filter(state.items, { activityId });
-  return orderBy(activityComments, 'createdAt', 'desc');
+export const getComments = state => params => {
+  const comments = filter(state.items, params);
+  return orderBy(comments, 'createdAt', 'desc');
 };
 
-export const getUnseenComments = (state, _, rootState) => activity => {
-  const { items, seenByActivity } = state;
-  const { user } = rootState.auth;
-  const lastSeen = seenByActivity[activity.uid] || 0;
-  return filter(items, it =>
-    it.authorId !== user.id &&
-    it.activityId === activity.id &&
-    new Date(it.createdAt).getTime() > lastSeen);
+export const getUnseenComments = (state, _, { auth }) => (activity, ce = null) => {
+  const { seen, items } = state;
+  const lastSeen = seen.contentElement[ce?.uid] || seen.activity[activity.uid] || 0;
+  const options = { activity, ce, lastSeen, user: auth.user };
+  return filter(items, it => setConditions(it, options));
 };
+
+function setConditions(it, { user, lastSeen, activity, ce }) {
```

Comment:
Consider eliminating short names like ce here 😉 F definition shouldn't have parameter names like it.